Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -161,6 +161,33 @@ class CurrentUserUpdater: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: @escaping (Result<[BlockedUser], Error>) -> Void) {
+        apiClient.request(endpoint: .loadBlockedUsers()) {
+            switch $0 {
+            case let .success(payload):
+                var blockedUsers: Set<BlockedUserDTO> = []
```

Comment:
Safer if this is `Set<BlockedUser>` then DTOs are only accessed in the write closure (and it is sure that correct thread accesses it).

Reply:
Interesting, but how will we update the DTO?:
```
session.currentUser?.blockedUsers = blockedUsers
```
Left side is `Set<BlockedUserDTO>` when the right side, in this case, is `Set<BlockedUser>` 🤔